File: 1078-remove-outermost-parentheses/1078-remove-outermost-parentheses.py
class Solution(object):
    def removeOuterParentheses(self, s):
        """
        :type s: str
        :rtype: str
        """
        count = 0
        list = []
        for i in s:
            if i == "(":
                if(count>0):
                    list.append(i)
                count+=1
            elif i == ")":
                count-=1
                if(count>0):
                    list.append(i)
        return "".join(list)

        

File: 1078-remove-outermost-parentheses/1078-remove-outermost-parentheses.java
class Solution {
    public String removeOuterParentheses(String s) {
        StringBuilder str = new StringBuilder();
        int count = 0;
        for(int i=0;i<s.length();i++){
            char ch = s.charAt(i);
            if(ch == '('){
                if(count>0){
                    str.append(ch);
                }
                count++;
            }else if(ch==')'){
                count--;
                if(count>0){
                    str.append(ch);
                }
            }
        }return str.toString();
    }
}